influxdb/client.py

```diff
@@ -145,10 +147,16 @@ def __init__(self,
             self._port,
             self._path)
 
-        self._headers = {
-            'Content-Type': 'application/json',
-            'Accept': 'application/x-msgpack'
-        }
+        if use_msgpack:
+            self._headers = {
+                'Content-Type': 'application/json',
+                'Accept': 'application/x-msgpack'
+            }
+        else:
+            self._headers = {
+                'Content-Type': 'application/json',
+                'Accept': 'text/plain'
```

Comment:
Why `text/plain` and not `application/json` ?

Reply:
You would have to ask the original implementor - this is the way it was implemented before moving the default to msgpack.